hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHomeMetrics.java

```diff
@@ -203,7 +203,19 @@ public enum PhoneHomeMetrics {
     REST_CONFIG_RELOAD_FAILURE("restconfigreloadfail"),
 
     REST_REQUEST_COUNT("restrequestct"),
-    REST_UNIQUE_REQUEST_COUNT("restuniqrequestct");
+    REST_UNIQUE_REQUEST_COUNT("restuniqrequestct"),
+
+    // DMT metrics
+    DMT_ENABLED("dmte"),
+    DMT_TIMESTAMP("t"),
+    DMT_NUMBER_OF_DS_TO_MIGRATE("dscnt"),
+    DMT_NUMBER_OF_DS_MIGRATED("dsmigcnt"),
+    DMT_MIGRATION_COMPLETED("cmp"),
+    DMT_SOURCE_VERSION("sv"),
+    DMT_TARGET_VERSION("tv"),
+    DMT_TARGET_VIRIDIAN("tvrd"),
+    DMT_DS_INFO("dsdata");
```

Comment:
I know that current phone home solution uses these short names, but do we need to? It's ugly and not much readable.

Reply:
Changed in https://github.com/hazelcast/hazelcast/pull/25715/commits/f3314a87a768bdfc51f39d658624c5089855e08b

let me know if you want to change some name to something else.